### ISSUE TITLE
PID: the K_p parameter does not effect the contro

### ISSUE THREAD

**Issue by Myzhar** [officinerobotiche/uNAV.X#37]
Trying to solve issue #36 I faced a problem in the PID controller: the K_p parameter is not enough to control the system.

I explain better. K_p, K_i and K_d are "fractional" variables, so they vary in the range [-1, 1). This means that K_p cannot be more then 0.9999999.

Trying to tune the PID I noticed that the control variable with K_p = 0.9999 (the max value possible) starts to move the motor only when the setpoint is more than 5 rad/s. 
Increasing the value of the setpoint, the control value increases, but the motor speed never reaches the setpoint.

So the system (MyzharBot in this case) is not reachable using a simple Proportional control. 
If I add an integral contribute, also really little I can control the motor.

I think that adding a 10x factor to K_p parameter would be useful, but I can do this only adding a gain on the error branch.

@rbonghi what do you think about this? And you @guiott ?

**Comment by Myzhar** [comment]
Another "problem" is related to the combination of the three parameters.

According to page what is written at page 74 of this document:
http://ww1.microchip.com/downloads/en/DeviceDoc/16bit_Language_Tool_Libraries_51456c.pdf

K_p + K_i + K_d should always be minor than 1, otherwise the coefficient abcCoefficients[0] will saturate.
At the same way Kp + 2*Kd should always be minor than 1, otherwise the coefficient abcCoefficients[1] will saturate.

In the PID Tuner application I inserted the controls. But I noticed that neither @rbonghi, neither @guiott verify these constrains in you firmware...
